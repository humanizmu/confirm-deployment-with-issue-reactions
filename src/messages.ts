export const getMissingGithubTokenMessage =
  () => `GitHub token is required, please provide token in the action input, example usage:
steps:
- uses: humanizmu/confirm-deployment-with-issue-reactions@some-version
  with:
    githubToken: \${{ secrets.GITHUB_TOKEN }}
`
export const getIssueUrlMessage = (
  issueUrl: string
) => `Opened issue ${issueUrl}
Confirm this step by reacting to the issue with 👍
To cancel this step react to the issue with 👎
`
