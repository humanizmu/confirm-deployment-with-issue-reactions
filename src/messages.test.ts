import {missingGithubTokenMessage} from './messages'

describe('messages', () => {
  describe('missingGithubTokenMessage()', () => {
    it('returns correct message', () => {
      expect(missingGithubTokenMessage()).toMatchSnapshot()
    })
  })
})
