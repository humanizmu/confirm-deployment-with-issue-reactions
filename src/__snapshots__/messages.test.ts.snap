// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`messages issueUrlMessage returns correct message 1`] = `
"Opened issue http://some-issue-url
Confirm this step by reacting to the issue with 👍
To cancel this step react to the issue with 👎
"
`;

exports[`messages missingGithubTokenMessage() returns correct message 1`] = `
"GitHub token is required, please provide token in the action input, example usage:
steps:
- uses: humanizmu/confirm-deployment-with-issue-reactions@some-version
  with:
    githubToken: \${{ secrets.GITHUB_TOKEN }}
"
`;
